src/core/include/openvino/core/extension.hpp

```diff
@@ -28,24 +28,28 @@ class OPENVINO_API Extension {
 
     virtual ~Extension();
 };
+}  // namespace ov
 
+#ifndef IE_CREATE_EXTENSION
 /**
  * @brief The entry point for library with OpenVINO extensions
  *
  * @param vector of extensions
  */
 OPENVINO_EXTENSION_C_API
-void create_extensions(std::vector<Extension::Ptr>&);
+void create_extensions(std::vector<ov::Extension::Ptr>&);
 
-}  // namespace ov
+#    define IE_CREATE_EXTENSION create_extensions
```

Comment:
```suggestion
#    define OV_CREATE_EXTENSION create_extensions
```

Reply:
This change will be moved to another PR, which focus on upgrade the plugin & extension create func definition.